### ISSUE TITLE
Suggestion: Eventually add a monospaced variant

### ISSUE THREAD

**Issue by thejupitergroup** [Corne2Plum3/Giphurs#14]
The Giphurs project is awesome, it is a clean and brilliant geometric typeface with numerous alternatives. A monospaced variant in the same design would look excellent.

**Comment by Corne2Plum3** [comment]
I agree with it and what I wanted to do next. But if possible I'd like to have this font ready or close to be uploadable on Google fonts so I have a solid base to draw the monospaced glyphs.

And about when, I don't know...
